include-build/roborazzi-core/src/commonMain/kotlin/com/github/takahirom/roborazzi/CaptureResult.kt

```diff
@@ -14,19 +15,17 @@ import kotlinx.serialization.json.JsonDecoder
 import kotlinx.serialization.json.decodeFromJsonElement
 import kotlinx.serialization.json.jsonObject
 import kotlinx.serialization.json.jsonPrimitive
-import java.io.File
-import java.io.FileReader
 
 @Serializable(with = CaptureResult.CaptureResultSerializer::class)
 sealed interface CaptureResult {
   val type: String
   val timestampNs: Long
-  val compareFile: File?
-  val actualFile: File?
-  val goldenFile: File?
+  val compareFile: Path?
+  val actualFile: Path?
+  val goldenFile: Path?
```

Comment:
@takahirom I was thinking of using `String` here to abstract away the `Path` class. That way we can decouple the use of the kotlinio package even further. Wdyt?

Reply:
As JSON is simply a string, I think it's appropriate to use a string here. I don't have a strong opinion about this.

Reply:
Thanks for your input. I ran away with your comment and went ahead and change it to `String` to abstract `kotlinx-io` class even away from the core code base